svgeditor/scripts.js

```diff
@@ -0,0 +1,97 @@
+// Order of operations
+// 1. Create UI
+// 2. Draw first SVG
+// 3. Event listener actions:
+// 3a. Create event listener
+// 3b. Update SVG
+// 3c. Redraw SVG
```

Comment:
So I'm on pc and the current path doesn't load the SVG to an object. But if I make it "assets/sampleSVG.svg" it loads right. Is this a platform thing?

Reply:
Also got it to work with "./assets/sampleSVG.svg"